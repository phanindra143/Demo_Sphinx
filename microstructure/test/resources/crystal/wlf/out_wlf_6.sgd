*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   171.3257
  !k_ij(1,2)=    -0.0479
  !k_ij(2,2)=   171.2803
  !k_ij(1,3)=     0.1321
  !k_ij(2,3)=     0.0332
  !k_ij(3,3)=   171.2383
MP,Kxx,1,  171.32569
MP,Kyy,1,  171.28026
MP,Kzz,1,  171.23828
MP,Kxx,2,  171.28026
MP,Kyy,2,  171.32569
MP,Kzz,2,  171.23828
MP,Kxx,3,  171.28026
MP,Kyy,3,  171.32569
MP,Kzz,3,  171.23828
MP,Kxx,4,  171.32569
MP,Kyy,4,  171.28026
MP,Kzz,4,  171.23828
MP,Kxx,5,  171.32569
MP,Kyy,5,  171.28026
MP,Kzz,5,  171.23828
MP,Kxx,6,  171.28026
MP,Kyy,6,  171.32569
MP,Kzz,6,  171.23828
MP,Kxx,7,  171.28026
MP,Kyy,7,  171.32569
MP,Kzz,7,  171.23828
MP,Kxx,8,  171.32569
MP,Kyy,8,  171.28026
MP,Kzz,8,  171.23828
MP,Kxx,9,  171.23828
MP,Kyy,9,  171.32569
MP,Kzz,9,  171.28026
MP,Kxx,10,  171.32569
MP,Kyy,10,  171.23828
MP,Kzz,10,  171.28026
MP,Kxx,11,  171.32569
MP,Kyy,11,  171.23828
MP,Kzz,11,  171.28026
MP,Kxx,12,  171.23828
MP,Kyy,12,  171.32569
MP,Kzz,12,  171.28026
MP,Kxx,13,  171.23828
MP,Kyy,13,  171.32569
MP,Kzz,13,  171.28026
MP,Kxx,14,  171.32569
MP,Kyy,14,  171.23828
MP,Kzz,14,  171.28026
MP,Kxx,15,  171.32569
MP,Kyy,15,  171.23828
MP,Kzz,15,  171.28026
MP,Kxx,16,  171.23828
MP,Kyy,16,  171.32569
MP,Kzz,16,  171.28026
MP,Kxx,17,  171.28026
MP,Kyy,17,  171.23828
MP,Kzz,17,  171.32569
MP,Kxx,18,  171.23828
MP,Kyy,18,  171.28026
MP,Kzz,18,  171.32569
MP,Kxx,19,  171.23828
MP,Kyy,19,  171.28026
MP,Kzz,19,  171.32569
MP,Kxx,20,  171.28026
MP,Kyy,20,  171.23828
MP,Kzz,20,  171.32569
MP,Kxx,21,  171.28026
MP,Kyy,21,  171.23828
MP,Kzz,21,  171.32569
MP,Kxx,22,  171.23828
MP,Kyy,22,  171.28026
MP,Kzz,22,  171.32569
MP,Kxx,23,  171.23828
MP,Kyy,23,  171.28026
MP,Kzz,23,  171.32569
MP,Kxx,24,  171.28026
MP,Kyy,24,  171.23828
MP,Kzz,24,  171.32569
*endif
