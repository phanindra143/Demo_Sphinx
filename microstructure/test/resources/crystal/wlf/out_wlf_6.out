            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\Rein_ohne_Poren\90-10
             3.00000      -2676968.00996          -936.65333         -7788.17557
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000             9.33189            33.00940
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000         -1580.44316      -2676254.02152         -4949.16015
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000            14.06540         15625.00000            26.22991
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000         -7888.28523         -6074.88946      -2675603.70905
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000            34.42326            32.08217         15625.00000
  /%
